src/networkAddresses.ts

```diff
@@ -109,15 +118,24 @@ export const NETWORK_ADDRESSES = {
     RESILIENT_ORACLE: "0x8f3618c4F0183e14A218782c116fb2438571dAC9",
     VTREASURY: "0xDDc9017F3073aa53a4A8535163b0bf7311F72C52",
     BINANCE_ORACLE: "0xB09EC9B628d04E1287216Aa3e2432291f50F9588",
+    ENDPOINT: "0xb6319cC6c8c27A8F5dAF0dD3DF91EA35C4720dd7",
+    LZ_LIBRARY: "0x38dE71124f7a447a01D67945a51eDcE9FF491251",
+    OMNICHAIN_GOVERNANCE_EXECUTOR: "",
   },
   arbitrumsepolia: {
     NORMAL_TIMELOCK: "0x1426A5Ae009c4443188DA8793751024E358A61C2", // arbitrumsepolia Multisig
     GUARDIAN: "0x1426A5Ae009c4443188DA8793751024E358A61C2",
     CHAINLINK_ORACLE: "0xeDd02c7FfA31490b4107e8f2c25e9198a04F9E45",
     RESILIENT_ORACLE: "0x6708bAd042916B47311c8078b29d7f432342102F",
+    ENDPOINT: "0x6098e96a28E02f27B1e6BD381f870F1C8Bd169d3",
+    LZ_LIBRARY: "0x88866E5A296FffA511EF8011CB1BBd4d01Cd094F",
+    OMNICHAIN_GOVERNANCE_EXECUTOR: "0x1Ec8CF3203EA972C221fC0AbC6062431aA210AeD",
   },
   arbitrumone: {
     NORMAL_TIMELOCK: "0x14e0E151b33f9802b3e75b621c1457afc44DcAA0", // arbitrumone Multisig
     GUARDIAN: "0x14e0E151b33f9802b3e75b621c1457afc44DcAA0",
+    ENDPOINT: "0x3c2269811836af69497E5F486A85D7316753cf62",
+    LZ_LIBRARY: "",
```

Comment:
0x4D73AdB72bC3DD368966edD0f0b2148401A178E2, according to https://docs.layerzero.network/v1/developers/technical-reference/mainnet/ultralightnodev2-and-noncecontract-addresses#arbitrum

Reply:
Although we are using default library, but I would fetch the library address using [this](https://arbiscan.io/address/0x3c2269811836af69497E5F486A85D7316753cf62#readContract#F13) function which require UA address i.e. of `OmnichainGovernanceExecutor`. Therefore, we can add this address when configuring this VIP for Arbitrum.